### ISSUE TITLE
matchmaking

### ISSUE THREAD

**Issue by madmas9966** [ValveSoftware/Dota2-Gameplay#22383]
hey value when will u fix matchmaking bug ? !!!!!!!!!!!!!!

**Comment by pcrivatz** [comment]
Agree, at least 2 weeks now, i want to play on a server i chose, not auto region.

**Comment by sydrvxd** [comment]
![image](https://github.com/user-attachments/assets/357480ee-2b13-4717-8440-c7b1136841ec)

this is really a pain

**Comment by mattcleigh** [comment]
![image](https://github.com/user-attachments/assets/286764cc-bd3b-4afc-9c7b-0aedafbcfa5d)

**Comment by madmas9966** [comment]
Zero respect for players

**Comment by WarPigxx** [comment]
Same, takes about 20 minutes to confirm the match then you play 25 minutes and wait another 20 to confirm the match. + when you try to cancel searching it gives you ban
